WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -71,6 +71,11 @@ struct WooShippingCreateLabelsView: View {
                     }
                 }
             }
+            .sheet(isPresented: $isOriginAddressListPresented) {
+                WooShippingOriginAddressListView(viewModel: viewModel.originAddresses)
+                    .presentationDetents([.medium, .large])
+                    .presentationDragIndicator(.visible)
+            }
```

Comment:
Moving the sheet to the view body to make code more readable. I misplaced this since the last PR when I was breaking down subviews.